test/express-error.js

```diff
@@ -12,7 +12,7 @@ var root = __dirname + '/public',
 
 mkdirp.sync(root + '/emptyDir');
 
-var cases = require('./common-cases-error');
+var cases = require('./secret/common-cases-error');
```

Comment:
Is the name of this folder something dictated by tap@1.x or does it just ignore any subfolders?